packages/xrpl/test/integration/index.ts

```diff
@@ -1,46 +1,46 @@
 /* eslint-disable import/export -- Tells webpack which files exist. */
-export * from './transactions/signerListSet'
-export * from './transactions/payment'
-export * from './transactions/offerCreate'
-export * from './transactions/offerCancel'
-export * from './transactions/signerListSet'
-export * from './transactions/checkCancel'
-export * from './transactions/checkCash'
-export * from './transactions/checkCreate'
-export * from './transactions/depositPreauth'
-export * from './transactions/paymentChannelCreate'
-export * from './transactions/paymentChannelClaim'
-export * from './transactions/paymentChannelFund'
-export * from './transactions/trustSet'
+export * from './transactions/signerListSet.test'
```

Comment:
Can we just export everything from the `integration/transactions` folder instead of listing all of these by hand?

Reply:
@JST5000 that isn't possible technically speaking

Reply:
But I updated the missing files

Reply:
Ah, unfortunate - sounds good :)